hello/views.py

```diff
@@ -0,0 +1,22 @@
+from django.shortcuts import HttpResponse
+from django.shortcuts import render
+
+# Create your views here.
+
+def hello(request):
+    return HttpResponse("Hello, world")
+
+def Ilona(request):
```

Comment:
konwencja nazewnicza funkcji w pythonie to snake case (czyli moja_nazwa), w tym konkretnym przypadku def ilona(request):